src/lib/mina_base/account.ml

```diff
@@ -79,6 +82,10 @@ module Index = struct
 
     type value = Vector.t
 
+    let to_input x =
+      List.map x ~f:(fun (b : Boolean.var) -> ((b :> Field.Var.t), 1))
```

Comment:
Is this making a field out of each bool?

Reply:
Thanks.  Isn't this an expensive representation compared to, for example, combining all the bits into one field element?  Why this way here?

Reply:
It is more expensive, but we never use a 'packed' version of the index in the rest of the circuit, so it only really makes sense to pack it for this purpose, and it's simplest to let `Random_oracle_input` do that. (Concretely, we use the bits of the index to choose whether to go 'left' or 'right' at each branch of the merkle tree.)

Reply:
👍🏼